src/lib/pixel-select-drop-down/pixel-select-drop-down.tsx

```diff
@@ -299,6 +351,21 @@ export const PixelDropDown = React.forwardRef<HTMLDivElement, DropDownProps>(
                                   option.value === value ? 'selected' : ''
                                 }
                               >
+                                {isShowRadio && (
+                                  <Radio
+                                    type='radio'
+                                    checked={option.value === selectedOption}
+                                    onChange={() => handleRadioClick(option)}
+                                  />
+                                )}
+                                {isShowCheckbox && (
+                                  <Checkbox
+                                    type='checkbox'
+                                    checked={selectedOption.includes(option)}
+                                    onChange={() => handleCheckboxClick(option)}
+                                  />
```

Comment:
@masabharry Why you are using `selectedOption.includes(option)` just give `value.isChecked`  for checked

Reply:
@Saif ur Rehman
 Because we are using array in multi select option